src/actions/tomoAction.js

```diff
@@ -315,10 +315,8 @@ export const createNewGame = () => async (dispatch, getState) => {
   const factory = state.tomo.factory;
   const from = state.tomo.account;
   const GameArtifact = require('contracts/Game');
-  await factory.methods
-    .createGame()
-    .send({ from })
-    .then(async () => {
+  await factory.methods.createGame().send({ from }, async (e, r) => {
+    if (!e) {
```

Comment:
```suggestion
if (e) return;
```
code kiểu early return vậy trông sẽ đẹp hơn